slick.dataview.js

```diff
@@ -1204,7 +1204,7 @@
         return;
       }
 
-      var previousPagingInfo = $.extend(true, {}, getPagingInfo());
+      var previousPagingInfo = Slick.Utils.extend({}, getPagingInfo());
```

Comment:
does that really work this way, I mean don't we need to use `Object.assign` instead? I see another problem also, the first argument of `$.extend(true, ...)` is to make the copy a deep clone but we don't directly have that functionality in JS and you also didn't include any of that, I'm not sure if deep copy is really necessary here but I do see it as a difference compare to previous code

Reply:
Yes, the utils function makes a deep copy by default.